Add Private Testnet bundles generator (#3216)
keep-network/keep-core: infrastructure/eth-networks/private-testnet/bundles/bundle-guide.adoc

:toc: left
:toclevels: 3
:sectanchors: true
:sectids: true
:source-highlighter: rouge
:icons: font

= Keep Network Private Testnet Bundle

Use this bundle to setup Keep Network node running on Keep Network Private Testnet.
The Keep Network Private Testnet is running against the 
link:https://goerli.net/[Ethereum Görli Testnet].

This is a quickstart guide, for the full documentation please visit
link:https://docs.keep.network/run-keep-node.html[Run Keep Node documentation].

== Stake and Register

This bundle comes with a Staking Provider Ethereum Account that was already staked and
authorized, similar to what a Staker would do in the production.

IMPORTANT: Delivering the Staking Provider Account details in the bundle is a

> simplification for this bundle. In the production the Staking Provider will have

Reviewer: ```suggestion
simplification for testnet. On mainnet, the Staking Provider will have
```

address for the Staker. See <<#provider-account>> section.

[#provider-account]
=== Provider Account

> A Staking Provider is responsible for providing a Staker with a Provider Account

Author: Actually it was confusing to me when I was going over different resources. What's the correct naming: `Staking Provider Address/Account` or `Provider Address/Account`?

> A Staking Provider is responsible for providing a Staker with a Provider Account

Reviewer: Staking Provider Adress/Account

Account address where the stake should be delegated to.

The Staking Provider Account is controlled by the Staking Provider.

The Staking Provider account can be an Ethereum account managed by any kind of
a wallet, that can sign transactions (i.e. it doesn't have to be a Key File). 

=== Operator Account

The Operator Account is an Ethereum account that the Keep Client runs with. The
client requires an encrypted Ethereum Key File along with the Password for the
Operator Account to run.

The Operator Account is controlled by the Staking Provider.

The Staking Provider has to register an Operator Account address for the stake delegation
received to the Staking Provider Account.

To generate an Ethereum Account Key File you can use `geth account new` command.

[source,shell]
----
geth account new --keystore ./keystore
----

Keep the password used for the Key File encryption as it will
have to be passed to the Keep Client start command.

Once the Operator Account address is known it should be registered with a transaction
submitted from the Staking Provider Account, please refer to

> link:https://docs.keep.network/registration.html#register-operator[Register Operator]

Reviewer 2: Shouldn't it be:
```suggestion
link:https://docs.keep.network/development/registration.html#register-operator[Register Operator]
```
?

Author: No, the `registration.adoc` is under the `docs/` directory. See preview here https://docs.keep.network/docs-v2/registration.html

Reviewer 2: Right, sorry, my bad...
